recipes/swan/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+schema_version: 1
+
+context:
+  name: swan
+  version: 41.51
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://swanmodel.sourceforge.io/download/zip/swan4151.tar.gz
+  sha256: 5e30107af500cadf9d6d1b6076403cc85793fd4db113d6a487fede5f74f41f77
+
+build:
+  number: 0
+  skip: win
+
+requirements:
+  build:
+    - make
+    - ${{ stdlib('c') }}
+    - ${{ compiler('c') }}
+    - ${{ compiler('fortran') }}
+  host:
+    - perl
+    - openmpi
+    - netcdf-fortran
+  run:
+    - openmpi
+  run_exports:
+    - ${{ pin_subpackage(name, upper_bound='x') }}
```

Comment:
If there is a shared library, please test for its existence.

Reply:
I am not sure what makes you think there is such.
`meta.yaml` (v0) [used](https://github.com/conda-forge/staged-recipes/blob/fb6786c6c134a0f6dcfb3230f6558103105eecf5/recipes/swan/meta.yaml#L16) `- {{ pin_subpackage(name, max_pin='x') }}` 
conda-recipe-converter (rattler-build) converted it to `dependencies > run_exports >...`

Only the binaries are needed. No static libraries.

Reply:
https://github.com/conda-forge/staged-recipes/pull/28726#discussion_r1926689675
I think I understand! `run_exports` should be used only when the recipe produces a shared library! Makes sense!

In Bioconda it is mandatory for all recipes and I added it automatically...